README.adoc

```diff
@@ -7,7 +7,15 @@ image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]
 
-This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards.
+This library allows the control and configuration of the BQ24195 PMIC as part of the following Arduino boards:
+
+* The following MKR boards
+** MKR WIFI 1010 (SKU: ABX00023)
+** MKR FOX 1200 (SKU: ABX00014)
+** MKR GSM 1400 (SKU: ABX00018)
```

Comment:
```suggestion
** MKR GSM 1310 (SKU: MKRWAN1310WANT)
** MKR GSM 1400 (SKU: ABX00018)
```
The MKR GSM 1310 does have a BQ24196 PMIC:

https://docs.arduino.cc/hardware/mkr-wan-1310#resources